nxrm-ha/README.md

```diff
@@ -320,6 +321,244 @@ Azure Key Vault is disabled by default. If you would like to store your database
+      So we recommend to use this method:  [Alternative: link Kubernetes ServiceAccounts to IAM](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#kubernetes-sa-to-iam) 
+
+   * 2.a
+   Enable Workload Identity:
+
+        ```    
+        gcloud container clusters update <your-cluster-name> \
+          --zone <your-cluster-zone> \
+          --workload-pool=<your-project-id>.svc.id.goog
+        ```
+        ```
+         gcloud container node-pools update NODEPOOL_NAME \
+         --cluster=CLUSTER_NAME \
+         --zone=COMPUTE_ZONE \
+         --workload-metadata=GKE_METADATA
+         ```
+   * 2.b
+   Create a Kubernetes Service Account:
+    
+         ```
+         kubectl create serviceaccount <your-k8s-service-account>
+         ```
+   * 2.c
+      Create a Google Service Account:
+        ```
+        gcloud iam service-accounts create <your-gsa-name> 
+        ```
+   * 2.d
+      Grant required roles to your <your-gsa-name> account
+
+        ```
+         gcloud projects add-iam-policy-binding <your-project-id> \
+         --member "serviceAccount:<your-gsa-name>@<your-project-id>.iam.gserviceaccount.com" \
+         --role "roles/storage.admin"
+        
+        gcloud projects add-iam-policy-binding <your-project-id> \
+        --member="serviceAccount:<your-gsa-name>@<your-project-id>.iam.gserviceaccount.com" \
+        --role="roles/compute.viewer"
+        
+        // if your intent to use ESO   
+        gcloud projects add-iam-policy-binding <your-project-id> \
+        --member="serviceAccount:<your-gsa-name>@<your-project-id>.iam.gserviceaccount.com" \
+        --role="roles/secretmanager.admin"
+        
+        // if your intent to use ESO    
+        gcloud projects add-iam-policy-binding <your-project-id> \
+        --member="serviceAccount:<your-gsa-name>@<your-project-id>.iam.gserviceaccount.com" \
+        --role="roles/iam.serviceAccountTokenCreator"
+        
+        // if your intent to use Google Artifactory to store Nexus docker images for deploy
+        gcloud projects add-iam-policy-binding <your-project-id> \
+        --member="serviceAccount:<your-gsa-name>@<your-project-id>.iam.gserviceaccount.com" \
+        --role="roles/artifactregistry.admin"
+        ```
+   * 2.e
+      Allow the Kubernetes Service Account to Act As the Google Service Account:
+
+        ```
+        gcloud iam service-accounts add-iam-policy-binding <your-gsa-name>@<your-project-id>.iam.gserviceaccount.com \
+          --role roles/iam.workloadIdentityUser \
+          --member "serviceAccount:<your-project-id>.svc.id.goog[<your-namespace>/<your-k8s-service-account>]"
+        ```
+   * 2.f
+        Annotate the Kubernetes Service Account:
+    
+        ```
+            kubectl annotate serviceaccount <your-k8s-service-account> \
+            --namespace <your-namespace> \
+            iam.gke.io/gcp-service-account=<your-gsa-name>@<your-project-id>.iam.gserviceaccount.com
```

Comment:
In both AWS and Azure usages of the helm chart, it is possible to annotate the service account by including it in the values.yaml i.e. serviceAccount.annotations. Does this not work for GKE?

Reply:
yes, it's possible and it's described in the doc above.
Updated - I marked this step as optional and added a link to a section where we update service account in values.yaml